frontend/apps/marketing/tests/e2e/all-the-things.spec.ts

```diff
@@ -2,8 +2,31 @@ import {expect, Locator} from '@playwright/test';
 import {test} from './fixtures/base';
 import {EXPECTED_LOCALIZATION_STRINGS} from './config/i18n';
 import {AllTheThingsPage} from './pom/all-the-things';
+import AxeBuilder from '@axe-core/playwright';
 
 test.describe('All the things UI e2e test', () => {
+  test.describe('a11y', () => {
+    test('should have no accessibility violations', async ({page}) => {
+      const allTheThingsPage = new AllTheThingsPage(page, 'en-US');
+      await allTheThingsPage.goto();
+
+      const accessibilityScanResults = await new AxeBuilder({page}).analyze(); // 4
+
+      // Do not allow any more accessibility errors. If you fixed one, reduce the number below.
+      if (accessibilityScanResults.violations.length !== 2) {
+        // Log out the violations so we can fix them
+        // The current allowed violations are:
+        // 1. color contrast on overline
+        // 2. iframe no title
```

Comment:
This one will be fixed once I merge https://github.com/code-dot-org/code-dot-org/pull/64399, want to remove this line now?